src/main/java/software/amazon/awssdk/crt/utils/StringUtils.java

```diff
@@ -23,4 +23,32 @@ public static String join(CharSequence delimiter, Iterable<? extends CharSequenc
         }
         return sb.toString();
     }
+
+    /**
+     * Decode Base64 byte[] array into a String in pure Java for Android and Java 7+ support.
+     * Based on: https://stackoverflow.com/a/4265472
+    */
+    private final static char[] BASE64_ALPHABET = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/".toCharArray();
```

Comment:
Not ideal, but **currently** we support Android API level 24. This, unfortunately, means we cannot use the built-in `Base64` library, as it is supported from API level 26+. We also cannot use `javax.xml.bind.DatatypeConverter` because it was removed in Java 11+.

To work around this, I found a pure Java based solution that works for the purposes of the Java keystore, at least on Mac.

However, a better solution *might* be to just bump the minimum version of Android we support to API level 26, which is Android 8, and then we can use the built-in Java `Base64` library. [Android 8](https://en.wikipedia.org/wiki/Android_Oreo) is no longer supported and got its last update a year ago, but [only 83% use Android 8+](https://9to5google.com/2022/05/20/android-2022-distribution-numbers-chart/), so we would lose roughly 5% of the possibly supported Android base, so maybe not.

Regardless, the pure Java solution should work for now.